holmes/plugins/toolsets/__init__.py

```diff
@@ -122,6 +157,11 @@ def load_builtin_toolsets(dal: Optional[SupabaseDal] = None) -> List[Toolset]:
         toolset.type = ToolsetType.BUILTIN
         # dont' expose build-in toolsets path
         toolset.path = None
+        # Set enabled status based on is_default property
+        if hasattr(toolset, 'is_default'):
```

Comment:
the is_default was used in other use cases also in the saas. Can you elaborate why we need this changeand why azuremonitormetrics is mark as default=true?